server/playground/index.js

```diff
@@ -1,33 +1,16 @@
-const buildSchema = require("@aerogear/data-sync-gql-core");
 const { makeExecutableSchema } = require("graphql-tools");
 const { ApolloServer } = require("apollo-server-express");
-const { log } = require("../logger");
-const models = require("../models");
-
-async function connectDataSources(dataSources) {
-    log.info("Connecting data sources");
-    Object.keys(dataSources).forEach(async key => {
-        const dataSource = dataSources[key];
-        try {
-            await dataSource.connect();
-        } catch (error) {
-            log.error(`Error while connecting datasource with key ${key}`);
-            log.error(error);
-            throw (error);
-        }
-    });
-}
-
+const { Core } = require("@aerogear/data-sync-gql-core");
+const config = require("../config").postgresConfig;
 
 async function ApplyExecutableMiddleware(app) {
```

Comment:
first letter could be small as this is an function not an class (not your change, but let's improve it)